### DISCUSSION TITLE
Go to the File

### DISCUSSION THREAD

**Discussion by Neodevils** [community/community#19486] [Mobile] [answered]
GitHub mobile is currently missing "Go to file" property, which is definitely needed for repositories have a lot of files.

**Comment by candyho** [comment, marked as answer]
Hey, thanks for your feedback. I will add this to our backlog to keep track. Let say we have added the "go to file" property, would you expect to be able to edit the file on mobile or more looking to view files when you are on the go?

**Reply by Neodevils** [reply]
To be honest, editing files via GitHub Mobile would be pretty good for mobile developers. So we won't have to install an another app to push our files to GitHub or pull it from GitHub.

But still, going to file and viewing it would be pretty fast especially if it's "searching for given letters".
